storage/ndb/rest-server/rest-api-server/resources/testdbs/fixed/hopsworks.sql

```diff
@@ -180,6 +182,504 @@ VALUES
         '2022-06-14 10:27:03',
         '2022-06-14 10:27:03',
         'myapikey1',
-        999,
+        10000,
         0
     );
+
+CREATE TABLE `feature_store` (
+                                 `id` int(11) NOT NULL AUTO_INCREMENT,
+                                 `name` varchar(100) COLLATE latin1_general_cs NOT NULL,
+                                 `project_id` int(11) NOT NULL,
+                                 `created` timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+                                 `hive_db_id` bigint(20) NOT NULL,
+                                 PRIMARY KEY (`id`),
+                                 KEY `project_id` (`project_id`),
+                                 KEY `hive_db_id` (`hive_db_id`),
+                                --  CONSTRAINT `FK_368_663` FOREIGN KEY (`hive_db_id`) REFERENCES `metastore`.`DBS` (`DB_ID`) ON DELETE CASCADE ON UPDATE NO ACTION,
+                                 CONSTRAINT `FK_883_662` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster AUTO_INCREMENT=67 DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+-- CREATE TABLE `on_demand_feature_group` (
```

Comment:
if you are not planning to use these tables in the future then you can remove the commented-out tables